crates/rooch-key/src/keystore/account_keystore.rs

```diff
@@ -69,6 +74,55 @@ pub trait AccountKeystore {
         Ok(result)
     }
 
+    fn export_mnemonic_phrase(
+        &mut self,
+        password: Option<String>,
+    ) -> Result<String, anyhow::Error> {
+        // load mnemonic phrase from keystore
+        let mnemonic = self.get_mnemonic(password.clone())?;
+        let mnemonic_phrase = mnemonic.mnemonic_phrase;
+        Ok(mnemonic_phrase)
+    }
+
+    fn export_private_key(
+        &mut self,
+        mnemonic: MnemonicResult,
+        mnemonic_phrase: String,
+        address: RoochAddress,
+    ) -> Result<String, anyhow::Error> {
+        // match the index of input address from the mnemonic data's addresses
+        let account_index = match mnemonic
+            .mnemonic_data
+            .addresses
+            .iter()
+            .position(|&target_address| target_address == address)
+        {
+            Some(index) => index as u32,
+            None => {
+                return Err(anyhow::anyhow!(
+                    "Address {} not found in the mnemonic data",
+                    address
+                ));
+            }
+        };
+        // get derivation path and seed
+        let derivation_path = generate_derivation_path(account_index)?;
+        let seed = derive_seed_from_mnemonic(Some(mnemonic_phrase), None)?; // assume word length is none
+                                                                            // derive 32 length secp256k1 key pair
+        let sk = derive_bitcoin_private_key_from_path(seed.as_bytes(), derivation_path)?;
+        let sk_bytes = sk.secret.as_ref();
```

Comment:
We should unlock the private key via password and then export it. So, `export_private_key` does not need the mnemonic argument.

Reply:
> We should unlock the private key via password and then export it.

I think every operations now are unlocked by password before exporting it:

```zsh
rooch account export -a rooch1ptyrgaz5gp5mjnunydg8lnalg7x5zqrhshr4c5g2qdy2u3wf5glqcm57m2

Enter the keystore password:
```

> So, `export_private_key` does not need the mnemonic argument.

The password is used for encryption not for decryption. In order to derive a key, a password is not needed, so there's no password for that.

The mnemonic argument is used for generating the seed and derive the secret key for function `derive_bitcoin_private_key_from_path`. Is there any other cases should I use to remove the mnemonic argument to derive the key?

Reply:
Okay. I now see a function `decrypt_with_type` was added.